packages/quantic/cypress/page-objects/actions/action-perform-search.ts

```diff
@@ -1,5 +1,5 @@
 export const performSearch = () =>
-  cy.get('c-action-perform-search button').click();
+  cy.get('c-action-perform-search button').click({force: true});
```

Comment:
why are we setting `force: true` here?

Reply:
Cause when there is a component error, a modal dialog like you can see is displayed.
And in Cypress you can't click on something while a modal is open unless you set force to true.